src/components/TradingViewChart/index.tsx

```diff
@@ -181,28 +214,26 @@ function ProLiveChart({
 
   return (
     <ProLiveChartWrapper fullscreen={fullscreen} onClick={() => setFullscreen(false)} className={className}>
-      {loading && (
+      {(loading || loadingProp) && (
         <Loader>
           <AnimatedLoader />
         </Loader>
       )}
 
       {isMobile ? (
-        <Portal>
-          <MobileChart
-            id="mobile25235"
-            ref={newRef => setRef(newRef)}
-            onClick={(e: any) => {
-              e.stopPropagation()
-            }}
-            fullscreen={fullscreen}
-            loading={loading}
-          ></MobileChart>
-        </Portal>
+        <MobileChart
+          id="mobile25235"
+          ref={newRef => setRef(newRef)}
+          onClick={(e: any) => {
+            e.stopPropagation()
+          }}
+          fullscreen={fullscreen}
+          $loading={loading || loadingProp}
```

Comment:
why do you need to add $ here?

Reply:
you can check it here, just to prevent the prop appear in DOM node https://styled-components.com/docs/api#transient-props